### PR TITLE
feat(comp/console): resource based auth

### PR DESCRIPTION
**Pull request by urbanisierung**
## Description

- Components: Console
- New page: Cluster Settings
- New section in manage users about resource based auth
- Console Ticket: https://github.com/camunda-cloud/camunda-cloud-management-apps/issues/558
- Screenshots will be finalized as soon as the feature is live

@Ben-Sheppard please add details about rba (I've added a comment into the docs)

## When should this change go live?

@Ben-Sheppard when will the feature go live?

- [x] This change is not yet live and should not be merged until {ADD_DATE} (apply `hold` label or convert to draft PR)?
- [ ] There is no urgency with this change.
- [ ] This change or page is part of a marketing blog, conference talk, or something else on a schedule.
- [ ] This functionality is already available but undocumented.
- [ ] This is a bug fix or security concern.

## PR Checklist

- [ ] I have added changes to the relevant `/versioned_docs` directory, or they are not for an **already released version**.
- [x] I have added changes to the main `/docs` directory (aka `/next/`), or they are not for **future versions**.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by Ben-Sheppard** [comment]
> @Ben-Sheppard when will the feature go live?

The feature is soft launching with alpha4 but is planned for 8.3

**Comment by Ben-Sheppard** [comment]
@urbanisierung - I've added some more content to the RBA section whilst trying to match the general feel of the current console documentation, let me know if you think it should have more :)

**Comment by urbanisierung** [comment]
@Ben-Sheppard Thanks for the additions! I also added a further screenshot of an authorized resource.

**Comment by urbanisierung** [comment]
@christinaausley not super urgent, but would be great if you could already review the doc changes in this PR. screenshots will be replaced as soon as we launch the feature. thanks!

File: docs/components/console/manage-clusters/img/cluster-settings.png
Comment (christinaausley) [inline docs/components/console/manage-clusters/img/cluster-settings.png:1]: We do have the SHA snippet and "tim paid" in the screenshot. Could be best to remove the header here.

**Review by christinaausley — APPROVED** [review]
Technical review in latest commit, and one comment to discuss screenshots 👍

File: docs/components/console/manage-clusters/img/cluster-settings.png
Comment (urbanisierung) [reply docs/components/console/manage-clusters/img/cluster-settings.png:1]: Screenshots will be replaced as soon as we've launched the feature, so we can create screenshots from production environment. Thanks for the review!

**Comment by akeller** [comment]
@urbanisierung, should this be moved out of draft and merged? Or is it not in prod yet?

**Comment by urbanisierung** [comment]
> @urbanisierung, should this be moved out of draft and merged? Or is it not in prod yet?

It's not on prod yet. It will go live in the September Release. That's just the preparation :)

File: docs/components/console/manage-clusters/img/cluster-settings.png
Comment (urbanisierung) [reply docs/components/console/manage-clusters/img/cluster-settings.png:1]: Screenshots have been updated.

**Comment by urbanisierung** [comment]
@christinaausley I've updated the screenshots. The new parts are now available under version `Next`, so from my point of view this can be merged.

**Comment by akeller** [comment]
Merging to unblock other RBA docs PRs (context - https://camunda.slack.com/archives/C058V245PNE/p1694679086251999?thread_ts=1694630463.625619&cid=C058V245PNE)

**Review by akeller — APPROVED** [review]

**Pull request merged**